bin/GameIndex.dbf

```diff
@@ -1616,6 +1626,9 @@ Region = PAL-M5
 Serial = SCES-51176
 Name   = Disney's Treasure Planet
 Region = PAL-M4
+EETimingHack = 1 // Fixes hang before going ingame
+eeRoundMode = 0 // Fixes characters flying into sky
+eeClampMode = 3 // Required to load areas correctly
```

Comment:
I'm not sure these are all still required, there might be differences between regions. Needs confirmation.

Reply:
Yes, I will keep this for now, waiting for test :).